### ISSUE TITLE
Training set

### ISSUE THREAD

**Issue by austnle** [dayoonkwon/BioAge#36]
Hi Dayoon!

Hope you're doing well and thanks for sharing this package. I understand that for hd_calc, the training dataset is NHANES III with individuals 20-30 years of age, a hypothetically young, "healthy" cohort, however for the phenoage_nhanes and kdm_nhanes, I am having a hard time understanding the NHANES III training cohort used for these functions. Are these populations also assumed to be "healthy agers"?

Also, in your code to create a table:
`
table3 = table_ses(data,agevar,exposure = c("edu","annual_income","poverty_ratio"), label)

#pull table
table3$table
`
you mentioned that "Models included covariates for chronological age and sex. The original KDM Biological Age algorithm (left-most column) was projected onto data from NHANES 2007-2010 only because other NHANES IV waves did not include spirometry measurements. The original Levine Phenotypic Age algorithm (second column from left) was projected onto data from NHANES 1999-2010 and 2015-2018 only because the intervening waves did not include CRP measurements."

How did you specify model covariates and the projection data into the table function?
